front/src/app/api/logs/[id]/connectTag/route.ts

```diff
@@ -0,0 +1,16 @@
+import { LogsData } from "@/services/LogsData";
+import { NextResponse } from "next/server";
+
+export async function POST(
+    req: Request,
+    { params }: { params: { id: string } },
+) {
+    const json = await req.json();
+    const tagId = json.tagId;
+    try {
+        await LogsData.connectTag(params.id, tagId);
```

Comment:
Took me a while to work out what this meant - naming is tough :( Perhaps being more explicit with something like `connectTagToLog` is worth it? Not sure